old/lib/LedgerSMB/DBObject/TaxForm.pm

```diff
@@ -156,6 +158,12 @@ sub get_metadata
     @{$self->{countries}} = $self->call_dbmethod(
                 funcname => 'location_list_country'
     );
+    my %regions = %{Locale::CLDR
+                    ->new($self->{_user}->{language})
+                    ->all_regions};
+    foreach (@{$self->{countries}}) {
+      $_->{name} = $regions{$_->{short_name}}
+    }
```

Comment:
I see this pattern in multiple places. Would it be an idea to factor it out and put it somewhere central instead?

Reply:
It would be a great idea! I'm just wondering where would be the best place

Reply:
How about LedgerSMB::I18N ?

Reply:
Good! Fixing.